pkg/image/image.go

```diff
@@ -59,16 +62,22 @@ func NewUDF(cli cli.Commander) *UDF {
 
 // Create makes the required call to UDF to
 func (u *UDF) Create(release, channel, arch string, ver int) (path string, err error) {
-	tmpDirName, _ := u.cli.ExecCommand("mktemp -d")
-	tmpFileName := filepath.Join(tmpDirName, outputFileName)
+	tmpDirName, err := u.cli.ExecCommand("mktemp", "-d")
+	if err != nil {
+		return
+	}
+	tmpFileName := filepath.Join(strings.TrimSpace(tmpDirName), outputFileName)
```

Comment:
It was returning a \n at the end, maybe trimspace is too strong for just that :) 
Thanks!